pkg/awsvpc/sg_analysis.go

```diff
@@ -0,0 +1,228 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package awsvpc
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+
+	"github.com/np-guard/models/pkg/connection"
+	"github.com/np-guard/models/pkg/ipblock"
+	"github.com/np-guard/models/pkg/netp"
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/commonvpc"
+)
+
+type SpecificAnalyzer struct {
+	sgResource         *types.SecurityGroup
+	referencedIPblocks []*ipblock.IPBlock
+	sgMap              map[string]*commonvpc.SecurityGroup
+}
+
+func NewSpecificAnalyzer(sg *types.SecurityGroup) *SpecificAnalyzer {
+	res := &SpecificAnalyzer{sgResource: sg}
+	return res
+}
+
+func getAllConnSet() *connection.Set {
```

Comment:
Is this function really needed? Its a single line.
And if it is, does it belong here? there's nothing specific to `aws sg` in it

Reply:
removed